pkg/cache/application.go

```diff
@@ -60,6 +60,7 @@ type Application struct {
 	placeholderAsk             *si.Resource // total placeholder request for the app (all task groups)
 	placeholderTimeoutInSec    int64
 	schedulingStyle            string
+	requestOriginatingTask     interfaces.ManagedTask // Original Pod which creates the requests
```

Comment:
Can we simplify this name to just `originatingTask`? The current name is a mouthful...